proto/target/target.pb.go

```diff
@@ -16,8 +16,8 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.31.0
-// 	protoc        v4.24.0
+// 	protoc-gen-go v1.33.0
+// 	protoc        v3.21.12
```

Comment:
Does it matter that an older version of protoc was used to generate the protobufs?

Reply:
yeah. I tried to update the protobuf compiler and it seems that `3.21.12` is the latest version.
 ```
 protobuf-compiler is already the newest version (3.21.12-8.2).
Summary:
  Upgrading: 0, Installing: 0, Removing: 0, Not Upgrading: 0
```

Unless I am missing something there.